File: MODUL 2/C/PRAK203-2310817320006-Alya_Rosaan.c
#include <stdio.h>

int main(){
    //case 1
    float a,b,i,j,x,y;
    float c,d,k,l,w,z;
    printf("");
    scanf("%f %f %f %f %f %f", &a, &b, &i, &j, &x, &y);

    float jumlah = (a-b)*(i/j)-(x+y);

    printf("%.3f\n", jumlah);

    //case 2
    printf("");
    scanf("%f %f", &c, &d);
    printf("");
    scanf("%f %f", &k, &l);
    printf("");
    scanf("%f %f", &w, &z);
   
    float hasil = (c-d)*(k/l)-(w+z);

    printf("%.3f\n", hasil);

    return 0;
}

File: MODUL 2/PYTHON/PRAK203-2310817320006-Alya_Rosaan.py
def hasil(a,b,i,j,x,y):
    jumlah = (a-b)*(i/j)-(x+y)
    return jumlah

mynum = input("")
nilai = mynum.split()

a,b,i,j,x,y = map(float, nilai)

total = hasil(a,b,i,j,x,y)
print(f"{total:.3f}")

a,b  = map(float, input().split())
i,j = map(float, input().split())
x,y = map(float, input().split())

total = hasil(a,b,i,j,x,y)
print(f"{total:.3f}")



